lib/core/include/irods/msParam.h

```diff
@@ -195,6 +195,7 @@ int
 parseMspForPhyPathReg( msParam_t *inpParam, keyValPair_t *condInput );
 int
 parseMspForPosInt( msParam_t *inpParam );
+int msp_to_string(msParam_t* _inpParam, char** _outParam);
```

Comment:
Let's stick to snake case for function and parameter names.

_Don't forget to update the implementation to use the same casing. Keeps clang-tidy happy/quiet._